SpatialGDK/Source/SpatialGDKFunctionalTests/Private/TestMaps/SpatialRPCTimeoutMap.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "TestMaps/SpatialRPCTimeoutMap.h"
+#include "GameFramework/PlayerStart.h"
+#include "SpatialGDKFunctionalTests/SpatialGDK/SpatialTestRPCTimeout/SpatialTestRPCTimeout.h"
+#include "SpatialGDKFunctionalTests/SpatialGDK/SpatialTestRPCTimeout/SpatialTestRPCTimeoutGameMode.h"
+
+USpatialRPCTimeoutMap::USpatialRPCTimeoutMap()
+	: UGeneratedTestMap(EMapCategory::CI_PREMERGE, TEXT("SpatialRPCTimeoutMap"))
+{
+	// clang-format off
+	SetCustomConfig(TEXT("[/Script/SpatialGDK.SpatialGDKSettings]") LINE_TERMINATOR
+		TEXT("QueuedIncomingRPCWaitTime=0") LINE_TERMINATOR
+		TEXT("[/Script/UnrealEd.LevelEditorPlaySettings]") LINE_TERMINATOR
+		TEXT("PlayNumberOfClients=2"));
+	// clang-format on
+
+	EnableMultiProcess();
+}
+
+void USpatialRPCTimeoutMap::CreateCustomContentForMap()
+{
+	ULevel* CurrentLevel = World->GetCurrentLevel();
+	ASpatialWorldSettings* WorldSettings = CastChecked<ASpatialWorldSettings>(World->GetWorldSettings());
+	WorldSettings->DefaultGameMode = ASpatialTestRPCTimeoutGameMode::StaticClass();
+
+	// Add the test
+	FTransform Transform1 = FTransform::Identity;
+	Transform1.SetLocation(FVector(-300.f, -500.f, 200.f));
+
+	AddActorToLevel<APlayerStart>(CurrentLevel, Transform1);
```

Comment:
Hmm, the map generation includes a player start as default. Maybe not needed?
```suggestion
```

Reply:
Ah this migh need a comment, essentially what's happening here is there is two characters and I'm making sure they appear far enough apart from each other so that we can see them from the client windows (this just helps debugging if the test is failing, because we can see whether the material is loaded properly)